bcda/main.go

```diff
@@ -116,16 +159,43 @@ func jobStatus(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	jsonData, err := json.Marshal(job)
-	if err != nil {
-		log.Fatal(err)
-	}
+	switch job.Status {
+	case "Pending":
+		fallthrough
+	case "In Progress":
+		w.Header().Set("X-Progress", job.Status)
+		w.WriteHeader(http.StatusAccepted)
+	case "Failed":
+		http.Error(w, http.StatusText(500), 500)
+	case "Completed":
+		w.Header().Set("Content-Type", "application/json")
+
+		fi := fileItem{
```

Comment:
I understand this `fileItem` is a placeholder right now, but do you have any thoughts on the `Type` that we'll ultimately need to use here? 🤔

Reply:
Didn't have anything in mind. The specs say each file item has `type` and `url`, and optionally `count`: https://github.com/smart-on-fhir/fhir-bulk-data-docs/blob/master/export.md#response---complete-status Open to suggestions!